content/sensu-go/5.18/guides/generate-certificates.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Generate certificates for your Sensu installation"
+linkTitle: "Generate Certificates"
+description: "After you install Sensu, learn how to secure it with public key infrastructure (PKI). Set up PKI and generate the certificates you need to secure Sensu."
+weight: 165
+version: "5.18"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-5.18:
+    parent: guides
+---
+
+- [Public key infrastructure](#public-key-infrastructure-pki)
+- [Issue certificates](#issue-certificates)
+    - [Install TLS](#install-tls)
+    - [Create a Certificate Authority (CA)](#create-a-certificate-authority-ca)
+    - [Generate backend cluster certificates](#generate-backend-cluster-certificates)
+    - [Generate agent certificate](#generate-agent-certificate)
+- [Next step: Secure Sensu](#next-step-secure-sensu)
+
+If you are deploying Sensu for use outside of a local development environment, you should secure it using transport layer security (TLS).
+TLS uses encryption to provide security for communication between Sensu backends and agents as well as communication between human operators and the Sensu backend, such as web UI or sensuctl access.
+
+We recommend that you configure TLS for your backend clusters from the very beginning because reconfiguring an existing Sensu backend cluster from cleartext to TLS can be time-consuming.
```

Comment:
```suggestion
Because reconfiguring an existing Sensu deployment from cleartext to TLS can be time-consuming, we recommend that you configure TLS for your backend from the very beginning.
```